### DISCUSSION TITLE
Unable to upload new app to play store

### DISCUSSION THREAD

**Discussion by Gokulv617** [airsdk/Adobe-Runtime-Support#1131] [Q&A] [answered]
Today I created a new project. I uploaded my .aab file and then Google itself has created a new key(deployment_cert.der) which can be downloaded from the console in the App integrity section. I use a common .p12 certificate for all the games. But now it seems to get the following error from the console.

My .p12 certificate has a different SHA1 fingerprint and the one google generated in the console is different. Please help

> You uploaded an APK or Android App Bundle that is signed with a key that is also used to sign APKs that are delivered to users. Because you are enrolled in Play App Signing, you should sign your APK or Android App Bundle with a new key before you upload it.

**Please tell me what should I do next to upload my app to play store ?**

**Do I need to generate a .p12 certificate from the deployment_cert.der? if yes, please let me know the steps.**

![img_20210908(1)](https://user-images.githubusercontent.com/43432782/132460669-f33d81fb-2e4e-4a80-a6c9-6216833a5b4f.png)

**Comment by Gokulv617** [comment, marked as answer]
This link was useful. I created another project and this time I successfully uploaded the .aab file.

[https://stackoverflow.com/questions/54299273/uploading-android-app-bundle-to-google-play-console-key-signing-error](https://stackoverflow.com/questions/54299273/uploading-android-app-bundle-to-google-play-console-key-signing-error)

**Reply by marchbold** [reply]
What exactly in that link helped you, there appear to be a lot of valid suggestions?

**Reply by Gokulv617** [reply]
The below comment helped me
> I hit this error in our apps. We use the same upload key for all of our apps.
> 
> So when you create a new app, make sure you select the option Use an existing key that you sign an app with instead of the (Google recommended) option which is to send them an unique upload key.
> 
> The really annoying part is that this option can only be selected once, so we had to delete our app and recreate it.
